rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -8500,9 +8480,7 @@ void opcode_02f5(const ScriptArguments& args, const ScriptObject object, const S
 	@arg xOffset X offset
 */
 void opcode_02f6(const ScriptArguments& args, const ScriptFloat angle, ScriptFloat& xOffset) {
-	RW_UNIMPLEMENTED_OPCODE(0x02f6);
-	RW_UNUSED(angle);
-	RW_UNUSED(xOffset);
+	xOffset = glm::sin(angle);
```

Comment:
The opcode argument is in degrees, but glm::sin takes radians. You can use glm::radians to convert the argument to rads.